server/controllers/stock/core.js

```diff
@@ -816,10 +816,46 @@ function computeLotIndicators(inventories) {
   return flattenLots;
 }
 
+/**
+ * @function getOpeningStockBalance
+ *
+ * @description
+ * This function gets the opening stock balance for a date, depot
+ * and inventory combination by looking up the most recent value from
+ * the stock_movement_status table.  If none is found, it returns 0s.
+ */
+async function getOpeningStockBalance(date, depotUuid, inventoryUuid) {
+  const quantities = await db.exec(` 
+    SELECT sum_quantity AS quantity FROM stock_movement_status
+    WHERE date < ? AND depot_uuid = ? and inventory_uuid = ?
```

Comment:
Shouldn't that be:  date <=

Reply:
No, because we want to show the balance as of the most recent date.  The values from the current date will be displayed in the grid, so it will be double-counting if we put the equals.